tfhe/benches/core_crypto/pbs_bench.rs

```diff
@@ -24,6 +32,24 @@ const SHORTINT_BENCH_PARAMS_GAUSSIAN: [ClassicPBSParameters; 4] = [
     V1_0_PARAM_MESSAGE_4_CARRY_4_KS_PBS_GAUSSIAN_2M128,
 ];
 
+#[cfg(feature = "gpu")]
+const SHORTINT_MULTI_BIT_BENCH_PARAMS: [MultiBitPBSParameters; 4] = [
+    PARAM_GPU_MULTI_BIT_GROUP_3_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
```

Comment:
Since multi-bit is the reference for GPU we should probably have 1_1,  2_2 and 3_3 for TUniform & Gaussian no?